docs/api/Auth0.Core.Exceptions.ApiException.html

```diff
@@ -251,6 +251,51 @@ <h5 class="parameters">Parameters</h5>
         <td><span class="parametername">innerException</span></td>
         <td><p>The exception that is the cause of the current exception, or a null 
 reference if no inner exception is specified.</p>
+</td>
+      </tr>
+    </tbody>
+  </table>
+  <h3 id="methods">Methods
+  </h3>
+  <a id="Auth0_Core_Exceptions_ApiException_CreateSpecificExceptionAsync_" data-uid="Auth0.Core.Exceptions.ApiException.CreateSpecificExceptionAsync*"></a>
+  <h4 id="Auth0_Core_Exceptions_ApiException_CreateSpecificExceptionAsync_System_Net_Http_HttpResponseMessage_" data-uid="Auth0.Core.Exceptions.ApiException.CreateSpecificExceptionAsync(System.Net.Http.HttpResponseMessage)">CreateSpecificExceptionAsync(HttpResponseMessage)</h4>
+  <div class="markdown level1 summary"><p>Create an instance of the specific exception required for this unsuccessful <span class="xref">System.Net.Http.HttpResponseMessage</span>.</p>
+</div>
+  <div class="markdown level1 conceptual"></div>
+  <h5 class="decalaration">Declaration</h5>
+  <div class="codewrapper">
+    <pre><code class="lang-csharp hljs">public static Task&lt;ApiException&gt; CreateSpecificExceptionAsync(HttpResponseMessage response)</code></pre>
+  </div>
+  <h5 class="parameters">Parameters</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Name</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.Net.Http.HttpResponseMessage</span></td>
+        <td><span class="parametername">response</span></td>
+        <td><p><span class="xref">System.Net.Http.HttpResponseMessage</span> to parse for the correct exception.</p>
+</td>
+      </tr>
+    </tbody>
+  </table>
+  <h5 class="returns">Returns</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.Threading.Tasks.Task</span>&lt;<a class="xref" href="Auth0.Core.Exceptions.ApiException.html">ApiException</a>&gt;</td>
+        <td><p>An instance of a <a class="xref" href="Auth0.Core.Exceptions.ApiException.html">ApiException</a> subclass containing the appropriate exception for this response.</p>
```

Comment:
It looks like docs wasnt generated when this method got added https://github.com/auth0/auth0.net/commit/58dc202df1926d90d8b8bbfab050df63cf97e4ec